packages/query-devtools/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@tanstack/query-devtools",
+  "version": "5.0.0-alpha.23",
+  "description": "Developer tools to interact with and visualize the TanStack Query cache",
+  "author": "tannerlinsley",
+  "license": "MIT",
+  "repository": "tanstack/query",
+  "homepage": "https://tanstack.com/query",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "types": "./build/types/index.d.ts",
+  "main": "./build/umd/index.js",
+  "module": "./build/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./build/esm/index.js",
+      "require": "./build/umd/index.js"
+    }
+  },
+  "scripts": {
+    "clean": "rimraf ./build",
+    "test:eslint": "eslint --ext .ts,.tsx ./src",
+    "test:types": "tsc",
+    "test:lib": "vitest run --coverage",
+    "test:lib:dev": "pnpm run test:lib --watch",
+    "build:types": "tsc --build"
+  },
+  "files": [
+    "build",
+    "src"
+  ],
+  "devDependencies": {
+    "vite-plugin-solid": "^2.5.0"
+  },
+  "dependencies": {
+    "@tanstack/query-core": "workspace:*",
```

Comment:
I'm not sure how this works with dependencies and peerDependencies. Does this not mean that the query-devtools will bring their own version of the query-core?

Reply:
I guess this is fine for now since we only use the types from query core so nothing from `query-core` is bundled in the `query-devtools` build. But I'll try to add it to `peerDependencies` and see if that breaks anything! `peerDependencies` would make more sense here. Also will update the scripts/config.ts in a bit!